mysql/const.go

```diff
@@ -30,10 +30,19 @@ var (
 	// TiDBReleaseVersion is initialized by (git describe --tags) in Makefile.
 	TiDBReleaseVersion = "None"
 
+	// CustomizedReleaseVersion is initialized by (your definition) in Makefile.
+	CustomizedReleaseVersion = "None"
+
 	// ServerVersion is the version information of this tidb-server in MySQL's format.
-	ServerVersion = fmt.Sprintf("5.7.25-TiDB-%s", TiDBReleaseVersion)
+	ServerVersion = fmt.Sprintf("5.7.25-%s", CustomizedReleaseVersion)
```

Comment:
Why not use `TiDBReleaseVersion` as the default value?

Reply:
refer the related TiDB pr to see what user want
```
select version()
UbiSQL-v1.0.0-rc1
```

Reply:
I mean, the default is the TiDB release version. After the user sets it, we change the value.